### PR TITLE
works is lookin good 🔥 

### PR DESCRIPTION
**Pull request by steppy452**
🔥 🔥 🔥 
image

### CODE REVIEW

**Comment by steppy452** [comment]
@jbalenti I could use some input on whether the order of things makes sense here - probably mostly in the "expanded" portion

File: app/src/Routes/Works/Works.js [hunk 1]
```diff
@@ -1,40 +1,125 @@
...
+    detailLink,
+    external
+  } = piece;
   const key = `${title}-${type}`;
   const allInstruments = instruments && instruments.join(', ');
+  const allMovements = movements && movements.join(', ');
+  const allComposerNotes = composerNotes && composerNotes.join(', ');
+  const hasMoreInfo = !!allInstruments || !!allMovements || !!allComposerNotes;
+
+  console.log('piece', piece);
```
Comment (myrcutio) [inline app/src/Routes/Works/Works.js:34]: left some debugging logs in

File: app/src/Routes/Works/Works.js [hunk 2]
```diff
@@ -1,40 +1,125 @@
...
+            {type && <div className={css.type}>({type})</div>}
+          </div>
+          <Typography variant="caption" color="textSecondary">
+            {publisher && `Publisher: ${publisher}`}
+            {publisher && duration && ` | `}
+            {duration && `Duration: ${duration}`}
+          </Typography>
+        </div>
+      </AccordionSummary>
+      <AccordionDetails>
```
Comment (myrcutio) [inline app/src/Routes/Works/Works.js:70]: accordion is a great idea, keeps the routing simple and works on mobile.

is there a way to start with it open if there's an anchor link?

Comment (steppy452) [reply app/src/Routes/Works/Works.js:118]: good call

**Pull request merged**